### PR TITLE
Image Style Props Resize Mode Supported

### PR DESCRIPTION
**Pull request by shefayaparwin**
1.  Supported all 5 resize modes ('cover', 'contain', 'stretch', 'repeat', 'center')
2.  Emitting onLoad & onError events

### CODE REVIEW

File: ReactSkia/components/RSkComponentImage.cpp [hunk 1]
```diff
@@ -26,33 +49,38 @@ void RSkComponentImage::OnPaint(
...
+/* Draw order 1. Background 2. Image 3. Border*/
     drawBackground(canvas,frame,imageBorderMetrics,imageProps.backgroundColor,imageProps.opacity);
-    RNS_PROFILE_START(drawImage)
-    sk_sp<SkImage> imageData=getImageData(path.c_str());
-    if(imageData) {
-      canvas->drawImageRect(imageData, rect, nullptr);
-    } else {
-      RNS_LOG_ERROR("Draw Image Failed for:" << path);
+    canvas->save();
+      /*TO DO: Handle filter quality based of configuration. Setting Low Filter Quality as default for now*/
```
Comment (Taym95) [inline ReactSkia/components/RSkComponentImage.cpp]: Do you need `filter quality` comment here also?

Comment (shefayaparwin) [reply ReactSkia/components/RSkComponentImage.cpp:74]: Comment here was a duplicate. Missed to notice. Will remove .

**Review by Kudo — APPROVED** [review]
LGTM 🚀

**Comment by Taym95** [comment]
LGTM 🚀

**Pull request merged**